src/init.cpp

```diff
@@ -1622,7 +1622,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
 
     // On first startup, warn on low block storage space
     if (!fReindex && !fReindexChainState && chain_active_height <= 1) {
-        uint64_t assumed_chain_bytes{chainparams.AssumedBlockchainSize() * 1024 * 1024 * 1024};
+        uint64_t assumed_chain_bytes{chainparams.AssumedBlockchainSize() * 1'000'000'000};
```

Comment:
```suggestion
        uint64_t assumed_chain_bytes{chainparams.AssumedBlockchainSize() * pow(2, 30)};
```

```suggestion
        uint64_t assumed_chain_bytes{chainparams.AssumedBlockchainSize() * 1024 * 1024 * 1024};
```

Storage multipliers are defined as powers of 2 in increments of 10. Either use `pow(2, 30)` or the original `1024 * 1024 * 1024` for correct GiB conversion.

Any of the above suggestions would work. I don't understand the change in math.